src/site/xdoc/index.xml

```diff
@@ -36,18 +36,20 @@
       <subsection name="Usage">
         <p>General instructions on how to use the Flatten Maven Plugin can be found on the <a href="usage.html">usage page</a>.</p>
         
-        <p>If you have questions regarding the plugin's usage, contact the <a href="mail-lists.html">user mailing list</a>. Posts to the mailing list are archived and could
+        <p>If you have questions regarding the plugin's usage, contact the <a href="https://www.mojohaus.org/flatten-maven-plugin/mailing-lists.html">user mailing list</a>.
```

Comment:
please use relative links - the same documentation

Reply:
Updated to mailing-lists.html.